src/store/reducers/authReducer.ts

```diff
@@ -0,0 +1,45 @@
+import {AuthAction, AuthState, SET_USER, SET_LOADING, SIGN_OUT, SET_ERROR, SET_SUCCESS} from '../types';
+
+const initialState: AuthState = {
+  user: null,
+  authenticated: false,
+  loading: false,
+  error: '',
+  needVerification: false,
+  success: '',
+};
+
+export default (state = initialState, action: AuthAction) => {
```

Comment:
Avoid export default an anonymous function.

Reply:
Not sure how to go about this particular issue.

Reply:
```typescript
const thing = () => {}

export default thing;  // This should always go to the last line of the file
```

Imma change the structure of the state management code so you don't need to fix this, but def take a note of the ↑.